### PR TITLE
Fix ADIF version tags

### PR DESCRIPTION
**Pull request by phl0**
@AndreasK79 had fixed one occurrence but there are more. Moreover I would like to propose not to include "version" in the version tag itself. Seems somewhat doubled :) Hope @AndreasK79 does not mind. If so I will apply his patch to the other places in the code.

Fixes https://github.com/magicbug/Cloudlog/issues/1837 and similar code parts.

### CODE REVIEW

File: application/views/adif/data/clublog.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Cloudlog ADIF export
-<ADIF_VER:5>3.1.2
+<ADIF_VER:5>3.1.4
 <PROGRAMID:<?php echo strlen($this->config->item('app_name')); ?>><?php echo $this->config->item('app_name')."\r\n"; ?>
-<PROGRAMVERSION:<?php echo strlen('Version ' . $this->config->item('app_version')); ?>>Version <?php echo $this->config->item('app_version')."\r\n"; ?>
+<PROGRAMVERSION:<?php echo strlen('Version ' . $this->config->item('app_version')); ?>><?php echo $this->config->item('app_version')."\r\n"; ?>
```
Comment (AndreasK79) [inline application/views/adif/data/clublog.php:4]: Need to remove version in strlen.

File: application/views/adif/data/exportall.php [hunk 2]
```diff
@@ -3,9 +3,9 @@
 	header('Content-Disposition: attachment; filename="'.$this->session->userdata('user_callsign').'-'.date('dmY-Hi').'.adi"')
 ?>
 Cloudlog ADIF export
-<ADIF_VER:5>3.1.2
+<ADIF_VER:5>3.1.4
 <PROGRAMID:<?php echo strlen($this->config->item('app_name')); ?>><?php echo $this->config->item('app_name')."\r\n"; ?>
-<PROGRAMVERSION:<?php echo strlen('Version ' . $this->config->item('app_version')); ?>>Version <?php echo $this->config->item('app_version')."\r\n"; ?>
+<PROGRAMVERSION:<?php echo strlen('Version ' . $this->config->item('app_version')); ?>><?php echo $this->config->item('app_version')."\r\n"; ?>
```
Comment (AndreasK79) [inline application/views/adif/data/exportall.php:8]: Need to remove version in strlen.

**Comment by AndreasK79** [comment]
@phl0 I don't mind at all. Just need to remove version in strlen.

File: application/views/adif/data/clublog.php (hunk 1 above)
Comment (phl0) [reply application/views/adif/data/clublog.php]: Tnx. Wasn't aware that this was present in the other files. Fixed now.

File: application/views/adif/data/exportall.php (hunk 2 above)
Comment (phl0) [reply application/views/adif/data/exportall.php]: Fixed.

**Comment by phl0** [comment]
Fixed the other files. Ready for merge.

**Comment by AndreasK79** [comment]
Tested adif-generating ok. Merge time.

**Pull request merged**